pkg/monitoring/schemas.go

```diff
@@ -160,10 +160,35 @@ const TransmissionAvroSchema = `
 }
 `
 
+const TelemetryAvroSchema = `
+{
+  "namespace": "link.chain.ocr2",
+  "type": "record",
+  "name": "telemetry_config_set",
+  "fields": [
+    {"name": "config_digest", "type": "string", "doc": "hexadecimal "},
+    {"name": "block_number", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "signers", "type": "string"},
+    {"name": "transmitters", "type": "string"},
+    {"name": "f", "type": "int", "doc": "uint8"},
+    {"name": "delta_progress", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "delta_resend", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "delta_round", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "delta_grace", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "delta_stage", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "r_max", "type": "long", "doc": "uint32"},
+    {"name": "s", "type": "string"},
+    {"name": "oracles", "type": "string"},
+    {"name": "feed_contract_status", "type": "string"}
+  ]
+}
```

Comment:
The way we encode schemas has changed in the main branch.
I'll push and update on the main branch!